recipes/libtins/build.bat

```diff
@@ -0,0 +1,11 @@
+mkdir build
```

Comment:
This should be named `bld.bat`. Then it will work.

Reply:
fixed.